Cakes-Co/style.css

```diff
@@ -1 +1,257 @@
 /*  Add your styling here */
+
+*{
+    margin: 0;
+    box-sizing: border-box;
+    padding: 0;
+}
+
+body{
+
+    
+    font-family: "Poppins";
+}
+
+.header{
+background-image: url("https://images.pexels.com/photos/7966383/pexels-photo-7966383.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" );
+background-position: center;
+background-size: cover;
+background-repeat: no-repeat;
+padding: 150px;
+position:relative;
+z-index: 2;
+text-align: center;
+}
+
+h1{
+    color: rgba(218, 213, 213, 0.979);
+    z-index: 3;
+    position: relative;
```

Comment:
As I see it's only needed for the main heading - `header-text`. It breaks the layout for the rest of the headings. Let me know!